Project/UtilResources/Constants.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Constants.swift
+//  Project
+//
+//  Created by Emmanuel Velez Agudelo on 31/10/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+import UIKit
+struct Constants {
+    public static var blueColor: UIColor = UIColor(hex:"00ADEE")!
+    public static var backgroundColor: UIColor = UIColor(red:0.92, green:0.96, blue:0.98, alpha:1.0)
+}
```

Comment:
These kind of 'constants' can be implemented in extensions, as explained before.